website/pages/en/arbitrum/l2-transfer-tools-faq.mdx

```diff
@@ -2,25 +2,35 @@
 title: L2 Transfer Tools FAQ
 ---
 
-## What are L2 Transfer Tools?
+### What are L2 Transfer Tools?
```

Comment:
This violates the `remark-lint-first-heading-level` rule:

>   5:1  error  First heading level should be `2`  remark-lint-first-heading-level

I'm assuming you changed it because you preferred the smaller font size visually? We previously discussed that [in Slack](https://graphprotocol.slack.com/archives/C0210QKF9HT/p1682444599052629), but we can reevaluate if you think this rule is too strict.

Reply:
Though I notice you have `##`s below, so maybe the fix here is to add a `##` above to categorize the first batch of questions?

Reply:
Ahh, I remember now. Been a minute. It should be fine the way it was! Thank you!

Reply:
Hmm, well, are you sure? Seems weird that some questions would be `##` and some would be `###`.